rdkit/Chem/AllChem.py

```diff
@@ -155,20 +155,21 @@ def GetConformerRMSMatrix(mol, atomIds=None, prealigned=False):
   # if necessary, align the conformers
   # Note: the reference conformer is always the first one
   rmsvals = []
+  confIds = [conf.GetId() for conf in mol.GetConformers()]
   if not prealigned:
     if atomIds:
       AlignMolConformers(mol, atomIds=atomIds, RMSlist=rmsvals)
     else:
       AlignMolConformers(mol, RMSlist=rmsvals)
   else:  # already prealigned
-    for i in range(1, mol.GetNumConformers()):
-      rmsvals.append(GetConformerRMS(mol, 0, i, atomIds=atomIds, prealigned=prealigned))
+    for i in range(1, len(confIds)):
```

Comment:
Why not iterate over `confIds[1:]` directly? I think it will add some clarity.

Reply:
On the other hand it adheres to the rest of the code, so I guess it's fine like it is. Although it might be marginally faster.

Reply:
I agree on that, but I did not want to change the code style.